examples/delegator/lib.rs

```diff
@@ -63,7 +63,7 @@ mod delegator {
                 .endowment(total_balance / 4)
                 .code_hash(accumulator_code_hash)
                 .salt_bytes(salt)
-                .instantiate()
+                .try_instantiate()
```

Comment:
If we are unwrapping anyways we should use `instantiate`. Otherwise this is setting a bad example.

Reply:
I agree I have made this change